quesma/quesma/schema_transformer.go

```diff
@@ -356,6 +356,9 @@ func (s *SchemaCheckPass) applyPhysicalFromExpression(currentSchema schema.Schem
 	if len(query.Indexes) == 1 {
 		if indexConf, ok := s.cfg[query.Indexes[0]]; ok {
 			useCommonTable = indexConf.UseCommonTable
+		} else {
+			// if index configuration is not found, use common table
+			useCommonTable = true
```

Comment:
Shouldn't this use `conf.UseCommonTableForWildcard` instead?

Reply:
:) Just wanted to avoid passing `conf` and open discussion

Reply:
(outside of scope of this PR, but even better if this code used the `IsCommonTable` from the `table_resolver`'s `Decision` - but this is probably a larger refactor)

Reply:
ok, I was thinking that this will require more boilerplate, so first wanted to have something working, but it seems that it's easy.